libraries/shared/watcher/storage_watcher.go

```diff
@@ -102,7 +104,11 @@ func (watcher StorageWatcher) transformDiffs() error {
 		case diff := <-diffs:
 			err := watcher.transformDiff(diff)
 			if err != nil {
-				logrus.Infof("error transforming diff: %s", err.Error())
+				if err == sql.ErrNoRows || reflect.TypeOf(err) == reflect.TypeOf(types.ErrKeyNotFound{}) {
```

Comment:
do we have to do this `reflect.TypeOf` stuff because `ErrKeyNotFound` is a custom error?

Reply:
It's because the initialized error includes details about the key that was not recognized, so the actual error that bubbles up doesn't == the type

Reply:
ah, gotcha. 🙏